src/Handlers/WebhookHandler.php

```diff
@@ -420,10 +419,33 @@ protected function commandPrefixes(): Collection
 
         return collect($prefixes)
             ->push('/')
-            ->map(fn (string $prefix) => str($prefix)->trim()->toString())
+            ->map(fn (string $prefix) => str($prefix)->trim())
             ->unique();
     }
 
+    protected function isCommand(Stringable $text, Collection $commandPrefixes): bool
+    {
+        $firstLetters = $commandPrefixes->map->substr(0, 1);
```

Comment:
@fabio-ivona I checked the code and found out that the result of executing the `commandPrefixes` method is used in the `startsWith` methods for Stringable on lines 117 and 402:

- https://github.com/defstudio/telegraph/blob/main/src/Handlers/WebhookHandler.php#L117
- https://github.com/defstudio/telegraph/blob/main/src/Handlers/WebhookHandler.php#L402

These methods allow you to work with an object of this type without manual conversions.

And I changed it only so that you could write it this way:

```php
$firstLetters = $commandPrefixes->map->substr(0, 1);
```
instead of:
```php
$firstLetters = $commandPrefixes->map(
    fn (string $name) => Str::substr($name, 0, 1)
);
```

Reply:
@andrey-helldar you're right, please ignore my previous comment